File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None):
    if len(array) < min_size:
        delta = min_size - len(array)
        while delta > 0:
            array.append(value)
            delta -= 1
    return(array)
#print(pad([1,2,3], 5))

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, value=null) => {
    //function pad(array, size, padding){	
        if (value === null){	
          value = 'None';	
        }	
        if (array.length < minSize){	
          let delta = (minSize - array.length);	
          while (delta > 0){	
            array.push(value);	
            delta --;	
          }	
        }	
        return array;	
}
console.log(pad([1,2,3], 3))